src/main.js

```diff
@@ -209,6 +225,16 @@ function changeCircleColor(element) {
   }
 };
 
+function changeColorBar(element) {
```

Comment:
This works!

Reply:
Have y'all heard of switch statements? Sometimes when you have an if-else statement where you're say, checking `currentActivity.category` for different values, you can write it in a cool, clean manner.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch